scripts/build/build.ps1

```diff
@@ -34,7 +34,18 @@ header "Compiling"
 _ "$RepoRoot\scripts\compile\compile.ps1" @("$Configuration")
 
 header "Running Tests"
-_ "$RepoRoot\scripts\test\runtests.ps1"
+# Capture PATH for later
```

Comment:
lol, fair :) We can still grab the PATH and DOTNET_HOME values in the cmd if needbe, but I like your solution better.